.gitignore

```diff
@@ -64,3 +64,5 @@ __pycache__/
 .idea
 output
 
+/.ptp-sync/
```

Comment:
Wondering why / . ?? typically its current directory and never /

Reply:
ptp-sync is a hidden folder. Ignores the folder in the repo root directory.